sql/libreehr_upgrade.sql

```diff
@@ -7,3 +7,5 @@ RENAME TABLE `openemr_postcalendar_limits` TO `libreehr_postcalendar_limits`;
 RENAME TABLE `openemr_postcalendar_topics` TO `libreehr_postcalendar_topics`;
 RENAME TABLE `openemr_session_info` TO `libreehr_session_info`;
 RENAME TABLE `openemr_module_vars` TO `libreehr_module_vars`;
+UPDATE `forms` SET `form_name` = 'Patient Encounter' WHERE `form_name` = 'New Patient Encounter';
+UPDATE `registry` SET `name` = 'Patient Encounter', `directory` = 'patient_encounter' WHERE `directory` = 'newpatient';
```

Comment:
Move to contribs?


On 2016-09-06 09:38 PM, Tony McCormick wrote:
>
> In sql/libreehr_upgrade.sql 
> <https://github.com/LibreEHR/LibreEHR/pull/149#discussion_r77748426>:
>
> > @@ -7,3 +7,5 @@ RENAME TABLE `openemr_postcalendar_limits` TO `libreehr_postcalendar_limits`;
> >  RENAME TABLE `openemr_postcalendar_topics` TO `libreehr_postcalendar_topics`;
> >  RENAME TABLE `openemr_session_info` TO `libreehr_session_info`;
> >  RENAME TABLE `openemr_module_vars` TO `libreehr_module_vars`;
> > +UPDATE `forms` SET `form_name` = 'Patient Encounter' WHERE `form_name` = 'New Patient Encounter';
> > +UPDATE `registry` SET `name` = 'Patient Encounter', `directory` = 'patient_encounter' WHERE `directory` = 'newpatient';
>
> Why are we bothering with "official" upgrade code? We should be 
> focused on NEW Install only. We have no users that need upgrades. 
> Having something unofficial in your tool kit for reusing test data is 
> fine, but don't make it part of the release
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/LibreEHR/LibreEHR/pull/149/files/52c73e6f8bd9a59efdc56fcff29497c2982dd941#r77748426>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AAhzF7HlABano9W_EasP9wnfVd8Z_IR2ks5qnhWzgaJpZM4JzTqH>.
>